docs/openapi.yaml

```diff
@@ -1,791 +1,1564 @@
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/Plan_Search_GET_200'
+        '401':
+          $ref: '#/components/responses/401'
         '403':
-          description: Request not made by an authorized user
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/403'
         '404':
-          description: Plan with the specified `planId` not found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/404'
         '500':
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/500'
 
-    delete:
-      summary: Remove a specific applied Plan
+  '/plan/{planId}/activity/{cursorPrimary}/{cursorSecondary}':
+    get:
+      summary: >
+        Fetch a Plan's activity, such as Comments and Reviews, retrieving
+        paginated results
       description: >
-        Deletes an existing applied Plan from the database.
-      operationId: deletePlanById
+        Returns an object containing a list of Plan's activity and the next
+        cursor to the pagination
+      operationId: getPlanActivity
       tags:
         - Plan
       parameters:
-        - name: planId
-          in: path
-          description: ID of the plan to remove
-          required: true
-          schema:
-            type: integer
-            format: int64
-          example: 72
+        - $ref: '#/components/parameters/Plan_GET_PlanId'
+        - $ref: '#/components/parameters/Search_CursorPrimary'
+        - $ref: '#/components/parameters/Search_CursorSecondary'
       responses:
-        '204':
-          description: Successfully deleted Plan
+        '200':
+          $ref: '#/components/responses/Plan_Activity_GET_200'
+        '401':
+          $ref: '#/components/responses/401'
         '403':
-          description: Request not made by an authorized user
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/403'
         '404':
-          description: Plan with the specified `planId` not found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/404'
         '500':
-          description: Internal server error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
+          $ref: '#/components/responses/500'
 
-  '/plan/{planId}/comments':
+  '/user/search/{text}/{role}/{cursorPrimary}/{cursorSecondary}':
```

Comment:
![Capture](https://user-images.githubusercontent.com/32185302/81131287-80fb9f00-8eff-11ea-9988-e8cf43d3a891.JPG)

Parameter examples look like they might be having some sort of reference / formatting issue in swagger.io.